python/GangaLHCb/test/GPI/TestExternalGaudiExec.py

```diff
@@ -110,6 +110,10 @@ def testSubmitJobComplete(self):
         from Ganga.GPI import jobs
 
         j = jobs[-1]
+        j = j.copy()
+        j.unprepare()
+        j.prepare()
+        j.submit()
```

Comment:
@rob-c No, but we have introduced another bug which it is not the job of this test to expose.